ui/imports/shared/views/AssetsView.qml

```diff
@@ -175,33 +215,42 @@ ColumnLayout {
         verticalPadding: 0
         contentWidth: availableWidth
 
+        StatusListView {
+            id: regularAssetsLV
+            Layout.fillWidth: true
+            Layout.preferredHeight: contentHeight
+            interactive: false
+            objectName: "assetViewStatusListView"
+            model: root.areAssetsLoading ? d.loadingItemsCount : customSFPM
+            delegate: delegateLoader
+            section {
+                property: "isCommunityAsset"
+                delegate: Loader {
+                    width: ListView.view.width
+                    required property string section
+                    sourceComponent: section === "true" ? sectionDelegate: null
+                }
+            }
+
+        }
+    }
+
+    Component {
+        id: sectionDelegate
         ColumnLayout {
-            width: parent.width
+            width: ListView.view.width
             spacing: 0
 
-            StatusListView {
-                id: regularAssetsLV
-                Layout.fillWidth: true
-                Layout.preferredHeight: contentHeight
-                interactive: false
-                objectName: "assetViewStatusListView"
-                model: CustomSFPM {}
-                delegate: delegateLoader
-            }
-
             StatusDialogDivider {
                 Layout.fillWidth: true
                 Layout.topMargin: Style.current.padding
                 Layout.bottomMargin: Style.current.halfPadding
-                visible: d.hasCommunityAssets
```

Comment:
This divider should still be visible iff there are some community assets

Reply:
as this is a part of the section divider already we dont need to have this condition anymore